proposals/0029-cooperative-vector.md

```diff
@@ -920,7 +927,7 @@ void ID3D12CommandList::ConvertLinearAlgebraMatrix(D3D12_LINEAR_ALGEBRA_MATRIX_C
 * If SrcLayout is row-major or column-major, then SrcStride should be greater than the length of a row/column, and a
   multiple of the element size.
 * If DestLayout is row-major or column-major, then DestStride should be greater than the length of a row/column, and a
-  multiple of the element size.
+  multiple of 16.
```

Comment:
Is it correct that both of these bullets are talking about row-major/column-major, but only dests have this stride requirement? Perhaps the 16 byte stride requirement should be unconditionally applied to the dest wording here, i.e. a new bullet that's not coupled to the row-major/column-major bit.

Reply:
The stride value is ignored for non row-major/column-major layouts, so it isn't really unconditionally applied to the dest.  Or am I misunderstanding your suggestion?

Reply:
Oh right, forgot that spec detail.